acceptance/tests/base/host/packages.rb

```diff
@@ -34,7 +11,7 @@ def get_host_pkg(host)
 
 step '#check_for_package : can determine if a package is installed'
 hosts.each do |host|
-  package = get_host_pkg(host)[0]
+  package = 'bash'
```

Comment:
This probably isn't correct for everything, but the old approach was getting invalid.

Reply:
at least for everything we test on, it should be fine.